internal/eventstore/eventstore.go

```diff
@@ -195,6 +195,10 @@ func (es *Eventstore) RegisterFilterEventMapper(eventType EventType, mapper func
 	return es
 }
 
+func EventData(event EventPusher) ([]byte, error) {
+	return eventData(event)
+}
+
```

Comment:
just wrapping a private func in the exact same public func does not make sense... please just make `eventData` public if needed